preload/preloader/preloader.go

```diff
@@ -141,18 +116,19 @@ func precertSubmitterJob(ctx context.Context, addedCerts chan<- *preload.AddedCe
 			log.Printf("Added precert chain for CN '%s', SCT: %s\n", c.Precert.TBSCertificate.Subject.CommonName, sct)
 		}
 	}
-	wg.Done()
 }
 
 func main() {
 	flag.Parse()
 	var sctFileWriter io.Writer
 	var err error
 	if *sctInputFile != "" {
-		sctFileWriter, err = os.Create(*sctInputFile)
+		sctFile, err := os.Create(*sctInputFile)
 		if err != nil {
-			log.Fatal(err)
+			log.Fatalf("Failed to create SCT file: %v", err)
```

Comment:
nit - these could probably be `Errorf` since they're in `main`

Reply:
Base Golang `log` doesn't have `Exitf` (which is what I believe you meant).

Reply:
I did indeed... :coffee: